libraries/SD_MMC/examples/SDMMC_Test/SDMMC_Test.ino

```diff
@@ -172,8 +210,34 @@ void testFileIO(fs::FS &fs, const char * path){
 
 void setup(){
     Serial.begin(115200);
-    if(!SD_MMC.begin()){
-        Serial.println("Card Mount Failed");
+    while(!Serial) { delay(10); }
+    Serial.println("SDMMC_Test.ino starting!");
+
+    // If you are using any other ESP32-S3 board than ESP32-S3-USB-OTG which has preset default pins, you will
+    //    need to specify the pins with the following example of SD_MMC.setPins()
+    // If you want to use only 1-bit mode, you can use the line with only one data pin (d0) begin changed.
+    // Please note that ESP32 does not allow pin change and will always fail.
+#if defined(SOC_SDMMC_USE_GPIO_MATRIX) && not defined(BOARD_HAS_SDMMC)
+    //if(! SD_MMC.setPins(clk, cmd, d0)){ // 1-bit line version
+    if(! SD_MMC.setPins(clk, cmd, d0, d1, d2, d3)){ // 4-bit line version
+        Serial.println("Pin change failed!");
+        return;
+    }
```

Comment:
Why not to use `use_1_bit_mode` so no need for user to comment/uncomment code :)
```
if(use_1_bit_mode) {
    if(!SD_MMC.setPins(clk, cmd, d0)) { 
         Serial.println("Pin change failed!");
         return;
    }
}
else {
    if(!SD_MMC.setPins(clk, cmd, d0, d1, d2, d3)) { 
         Serial.println("Pin change failed!");
         return;
    }
}
```